src/ansys/mapdl/core/krylov.py

```diff
@@ -0,0 +1,612 @@
+import os
+import weakref
+
+import numpy as np
+
+from .mapdl_grpc import MapdlGrpc
+
+
+class KrylovSolver:
+    """Abstract mapdl krylov class.  Created from a ``Mapdl`` instance.
+
+    Examples
+    --------
+    Create an instance.
+
+    >>> from ansys.mapdl.core import launch_mapdl
+    >>> mapdl = launch_mapdl()
+    >>> ....
+    >>> ....
+    >>> Generate the FEA model (mesh, constraints, loads, etc.)
+    >>> Generate the .full file
+
+    >>> mk = mapdl.krylov
+
+    Generate Krylov subspace
+
+    >>> Qz = mk.krygensub(10, 500, True, True)
+
+    Reduces system of equations and solve at each frequency
+
+    >>> Yz = mk.krysolve(0, 1000, 100, 0, True)
+
+    Expand reduced solution back to FE space
+
+    >>> res = mk.kryexpand(True, 3)
+
+    """
+
+    def __init__(self, mapdl):
+        if not isinstance(mapdl, MapdlGrpc):
```

Comment:
```suggestion
        if not isinstance(mapdl, MapdlGrpc):  # pragma: no cover
```

This makes the code coverage to not account for this line.